GWMiniGame/GWMiniGame.cpp

```diff
@@ -0,0 +1,287 @@
+#define _CRT_SECURE_NO_WARNINGS
+
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <string>
+#include <cstdlib>
+#include <ctime>
+
+char* wordInput() {
+	const int maxWordLength = 5;
+	char buffer[maxWordLength + 1];
+	std::cout << "ENTER : ";
+	std::cin >> buffer;
+	std::cout << std::endl;
+
+	char* result = new char[maxWordLength + 1];
+	for (int i = 0; i <= maxWordLength; ++i) {
+		result[i] = buffer[i];
+	}
+	return result;
+}
+
+int mainMenu(){
+	int value = 0;
+	std::cout << "Greetings in Guess Word Game!" << std::endl;
+	std::cout << "1 - Random Word" << std::endl;
+	std::cout << "2 - Word of the day" << std::endl;
+	std::cout << "0 - Exit" << std::endl;
+	std::cout << "Enter: ";
+	std::cin >> value;
+	std::cout << std::endl;
+	return value;
+}
+
+std::string randomWordFromBD(const char* fileName) {
+	std::fstream file;
+	file.open(fileName, std::ios::in);
+
+	std::string randomWord;
+
+	if (!file) {
+		std::cout << "File not found!\n";
+	}
+	else {
+		std::vector<std::string> words;
```

Comment:
it's somewhat wasteful to create std::vector and use just for tmp reading from the file. What about changing file format just a bit?
- Add number of lines in the file as the first entry;
- calculate random using this number of words;
- move file pointer directly to the randomed line (using some of the `seek` methods) and read only that instead of ALL lines.
It may be sowewhat of a hussle but still worth considering)